indico/modules/payment/plugins.py

```diff
@@ -93,6 +93,16 @@ def get_event_management_url(self, event, **kwargs):
         if session.user and self.can_be_modified(session.user, event):
             return url_for('payment.event_plugin_edit', event, method=re.sub(r'^payment_', '', self.name))
 
+    def get_invalid_regforms(self, event):
+        """Return registration forms with incompatible currencies"""
+        from indico.modules.events.registration.models.forms import RegistrationForm
+        invalid_regforms = []
+        if self.valid_currencies:
```

Comment:
`is not None` - right now having an empty set of valid currencies would indicate that it does not support any currency. And while this case is unlikely to ever happen I'd rather stay consistent